docs/template/public/less/_kss-content.less

```diff
@@ -0,0 +1,128 @@
+// shared styles across pages 
+#kss-content {
+    .sectionSpacing {
+        padding-bottom: 30px;
+        padding-top: 30px;
+    }
+    .table-behavoir-cell .seperator {
+        margin: 0 50px 0 0;
+    }
+
+    /* patterns */
+    .visual-code {
+        margin: 0;
+    }
+
+    h3.popover-title {
+        margin-bottom: 0;
+    }
+
+    .usercard-info p {
+        margin-bottom: 0;
+    }
+
+    .modal-dialog,.popover {
+        z-index: 0;
+    }
+
+    .circle {
+        border: 1px solid #cfd8e3;
+        box-shadow: 0px 1px 1px #fff;
+        margin: 0 auto 20px auto;
+        width: 100%;
+        height: 0;
+        padding: 50% 0;
+        border-radius: 50%;
+        
+    }
+
+    .circle-type {
+        font-size: 74px;
+        display: inline-block;
+        margin-top: -35%;
+    }
+
+    .seperator {
+        margin: 40px 0 80px 0;
+        position: relative;
+
+        .circle {
+            top: 38px;
+            left: 0;
+            position: relative;
+            width: 35px;
+            height: 35px;
+            background-color: #d8dfe8;
+            margin: auto;
+            display: block;
+            z-index: 2;
+            padding:0;
+            border: none;
+            box-shadow: none;
+
+            .glyphicon {
```

Comment:
this level is nested to deep.  Bring it up to the save level as the .circle class